pkg/scheduler/api/namespace_info.go

```diff
@@ -0,0 +1,153 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package api
+
+import (
+	"fmt"
+	"sync"
+
+	"github.com/golang/glog"
+
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/client-go/tools/cache"
+)
+
+const (
+	// QuotaKey is the key in ResourceQuota.spec.hard indicating the weight of thi namespace
+	QuotaKey = "volcano.sh/namespace.weight"
+	// DefaultWeight is the default weight of namespace
+	DefaultWeight = 1
+)
+
+// NamespaceInfo records information of namespace
+type NamespaceInfo struct {
+	Name string
```

Comment:
What's Name mean?

Reply:
`Name` is the name of this namespace, I'll add comments above it.

Reply:
move its type from string to `NamespaceName`